chapters/03_queueing_model/main.tex

```diff
@@ -0,0 +1,252 @@
+\section{Queueing theoretic model}
+
+One of the main outcomes of this research is the creation of a queueing network
+model that consists two waiting zones and accepts two types of individuals.
+
+\input{chapters/03_queueing_model/img/queueing_model_diagram.tex}
+
+The model consists of two types of individuals; type 1 and type 2.
+Type 1 individuals arrive instantly at waiting zone 1 and wait to receive their
+service.
+Type 2 individuals arrive at waiting zone 2 and wait there until they are
+allowed to move to waiting zone 1.
+They are allowed to proceed only when the number of
+individuals in waiting zone 1 \textbf{and} in service is less than a
+pre-determined threshold \(T\).
+When the number of individuals is equal to or exceeds this threshold, all
+type 2 individuals that arrive will stay \textit{blocked} in waiting zone 2
+until the number of people in waiting zone 1 falls below \(T\).
+This is shown diagrammatically in Figure \ref{fig:diagram_of_queueing_system}.
+The parameters of the described queueing model are:
+
+\begin{itemize}
+    \item \(\lambda_i\): The arrival rate of type \(i\) individuals where
+    \(i\in\{1, 2\}\)
+    \item \(\mu\): The service rate for individuals receiving service at
+    waiting zone 1
+    \item \(C\): The number of servers
+    \item \(T\): The threshold at which individuals of the second type are
+    blocked
+\end{itemize}
+
+In chapter ?
+% TODO: reference chapter \ref{ch:ed-ems-application}
+this queueing network will be used to model the emergent behaviour between
+Emergency Departments (EDs) and the Emergency Medical Systems (EMS).
+
+\subsection{Discrete Event Simulation}
+% TODO: Is this subsection needed here? What would even be here?
+
+\subsection{Markov chain model}
+
+A Markov chain is a stochastic model that is the primary analytical tool to 
+study queues.
+Under the assumption that all rates (arrival and service) are Markovian the
+queueing system can be represented by a Markov chain
+model~\cite{kemeny1976markov}.
+The states of the Markov chain are denoted by \((u,v)\) where:
+
+\begin{itemize}
+    \item \(u\) is the number of individuals blocked in waiting zone 2
+    \item \(v\) is the number of individuals either in waiting zone 1 or in the
+    service centre
+\end{itemize}
+
+The set of all possible combination of pairs \((u, v)\) form all the possible 
+states that the system can visit.
+We denote the state space of the Markov chain as the set \(S=S(T)\) which can be
+written as the disjoint union:
+
+\begin{align}
+    S(T) =& S_1(T) \cup S_2(T) \text{ where:} \nonumber \\
+    S_1(T) =& \left\{(0, v)\in\mathbb{N}_0^2 \; | \; v < T \right\}
+    \label{eq:definition_of_S_as_disjoint_union} \\
+    S_2(T) =& \{(u, v)\in\mathbb{N}_0^2 \; | \; v \geq T \} \nonumber
+\end{align}
```

Comment:
A diagram of the "pan handle" state space with circles whopping what is S_1 and what is S_2 could be nice here.

Reply:
I see you have